lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -14,6 +14,33 @@ include_once 'includes/CampaignTransformer.php';
 include_once 'dosomething_campaign.theme.inc';
 define('DOSOMETHING_CAMPAIGN_PIC_STEP_HEADER', 'Snap a Pic');
 
+function dosomething_campaign_init() {
```

Comment:
whooops missed this one. yeah, thats a good point as its more global logic than campaign logic.